Formula/libb64.rb

```diff
@@ -0,0 +1,34 @@
+class Libb64 < Formula
+  desc "Base64 encoding/decoding library"
+  homepage "https://libb64.sourceforge.io/"
+  url "https://downloads.sourceforge.net/project/libb64/libb64/libb64/libb64-1.2.1.zip"
+  sha256 "20106f0ba95cfd9c35a13c71206643e3fb3e46512df3e2efb2fdbf87116314b2"
+
+  def install
+    system "make", "all_src"
+    include.mkpath
```

Comment:
Are you sure this is necessary? Try without it.

Reply:
> @schloegl Please grant maintainers permission to push to your fork (checkbox on the right-hand side of the PR), thanks!

That checkbox was already selected, you should be able to push.